.githooks/pre-commit

```diff
@@ -24,6 +24,12 @@ declare -a descriptions=(
   "Regions embedded as part of resource descriptions"
 )
 
+FILE=*.secret
+count=`ls -1 *.secret 2>/dev/null | wc -l`
```

Comment:
I think you might have meant to refer to `"${FILE}"` here? Otherwise the var is a bit redundant :)

Reply:
D'oh.  Removed...